Scripts/Proxies/TouchProxy.cs

```diff
@@ -21,7 +21,8 @@ public override IEnumerator Start()
 #if UNITY_EDITOR
 			EditorApplication.delayCall += () =>
 			{
-				transform.localPosition = Vector3.zero;
+				if (this != null)
+					transform.localPosition = Vector3.zero;
```

Comment:
Do we need to do the same for ViveProxy?

Reply:
maybe !

Reply:
ViveProxy isn't setting transform position anywhere in its start function